saleor/graphql/core/mutations.py

```diff
@@ -609,6 +594,29 @@ def check_channel_permissions(
                 message="You don't have access to some objects' channel."
             )
 
+    @classmethod
+    def create_metadata_from_graphql_input(
+        cls, metadata_list: list[MetadataInput] | None, *, error_field_name: str
+    ) -> MetadataItemCollection:
+        """Wrap the creation of metadata and map the error to graphql error.
```

Comment:
> and map the error to graphql error.

IMHO, for me more important information here would be: this method can raise ValidationError. The conversion from metadata's error  is more like implementation detail.

Reply:
done